test/05-device-state.spec.coffee

```diff
@@ -47,6 +47,9 @@ testTarget1 = {
 			'RESIN_SUPERVISOR_POLL_INTERVAL': '60000'
 			'RESIN_SUPERVISOR_VPN_CONTROL': 'true'
 			'RESIN_SUPERVISOR_PERSISTENT_LOGGING': 'false'
+			'RESIN_SUPERVISOR_DNS_SERVERS': ''
```

Comment:
Is there a test for removing config vars?

Reply:
Also as a note I think it's impossible to set an empty string as the value in the ui